### PR TITLE
[ai creator] Setup AI Art Generator and packages

### PR DESCRIPTION
**Pull request by jamiecdavidson**
**Description**

1. Initial setup of AIArtGenerator.js file.
2. Add npm package file for deepai install.

**Trello**

- Link: https://trello.com/c/0ToqNr2r

**Extra Info**
This is a placeholder for code that is required to be developed. Packages required for deepai have been installed.

### CODE REVIEW

**Comment by jamiecdavidson** [comment]
[F4 - Digital image auto generated by AI models 7](https://trello.com/c/0ToqNr2r/58-f4-digital-image-auto-generated-by-ai-models-7)

File: ai-creator/AIArtGenerator.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const deepai = require('deepai');
+
+deepai.setApiKey('quickstart-QUdJIGlzIGNvbWluZy4uLi4K') //This key free (untested)
```
Comment (xuqun123) [inline ai-creator/AIArtGenerator.js]: Can we make this key to be dynamically passed in? 
Can we also create our own account and use the newly generated API key?

File: ai-creator/AIArtGenerator.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const deepai = require('deepai');
```
Comment (xuqun123) [inline ai-creator/AIArtGenerator.js]: Your VsCode Auto formatting doesn't seem to work. 

Can you make sure to install `eslint` and `prettier` extensions in VScode, and make sure your codes are auto formatted correctly?

**Comment by xuqun123** [comment]
Can we also move all these logics into the backend folder as we won't host our own AI model anymore.

This way it's currently only invoking public APIs to get AI art outputs, so i reckon all these codes need to be moved to the `/backend` folder.

If it's only for proof of concept, that's fine. We can leave it here. Feel free to merge in if that's the case.

**Review by xuqun123 — APPROVED** [review]

File: ai-creator/AIArtGenerator.js (hunk 1 above)
Comment (jamiecdavidson) [reply ai-creator/AIArtGenerator.js]: I will find out how to get our own key and I agree that we should pass this in dynamically.

File: ai-creator/AIArtGenerator.js (hunk 2 above)
Comment (jamiecdavidson) [reply ai-creator/AIArtGenerator.js]: I have then installed I will make sure they are being used correctly!

**Pull request merged**